src/api/app/views/webui2/webui/main/_system_status_legend.html.haml

```diff
@@ -0,0 +1,21 @@
+%p
+  The above graphs show the number of active build jobs last week, currently
```

Comment:
:thought_balloon:  It's not really above anymore :cowboy_hat_face:

Reply:
Ooops loll. I will change this.

Reply:
I improved the text a bit. It reads better now.